Source/WebCore/rendering/RenderBox.cpp

```diff
@@ -5302,44 +5302,16 @@ void RenderBox::flipForWritingMode(FloatRect& rect) const
         rect.setX(width() - rect.maxX());
 }
 
-LayoutPoint RenderBox::topLeftLocation() const
+LayoutPoint RenderBox::topLeftLocationWithFlipping() const
```

Comment:
We don't say "with flipping". We refer to "logical" and "physical" coordinates (though I see there is prior art).

Reply:
Agreed, the mixed terminology is confusing -- this function is used in other places where also 'flipForWritingMode' is relevant. IMHO this should be cleaned up simultaneously.

Reply:
However, not sure how relevant non-LFC related RenderBox cleanups still are?

Reply:
While it is a valid observation, I don't think we should stop making code cleanups because some project will render this code redundant at some point.